docs/safety-eval/safety.md

```diff
@@ -0,0 +1,101 @@
+
+## Running on an interactive session
+
+Run the following command from the main project dir `open-instruct`.
+### Safety benchmarks
+
+For all benchmarks requiring safety evaluation unless noted otherwise, as a default, we use the [WildGuard](https://github.com/allenai/wildguard) classifier to evaluate the safety of model outputs.
+
+- [WildGuardTest](https://arxiv.org/abs/2406.18495)
+- [Harmbench](https://arxiv.org/abs/2402.04249)
+- [ToxiGen](https://arxiv.org/abs/2203.09509)
+- [XSTest](https://arxiv.org/abs/2308.01263)
+- [JailbreakTrigger (in TrustLLM)](https://arxiv.org/abs/2401.05561)
+- [Do-anything-now](https://arxiv.org/abs/2308.03825)
+- [WildJailbreak](https://arxiv.org/abs/2406.18510) (both harmful and benign contrast sets)
+
+```commandline
+PYTHONPATH=safety-eval python safety-eval/evaluation/run_all_generation_benchmarks.py    \
```

Comment:
managed to run locally! But is there a way to use the chat template provided in the HF tokenizer? Currently thats what the rest of our eval suite does, and so having to specify a different thing here is a bit annoying.

Reply:
If this is easy, I can add a script to run the safety eval automatically with the rest of the eval suite. If not, I say we just leave this PR here for now in case people want to run safety evals, and otherwise wait for oe-eval.

Reply:
also have some small fixes to the safety fork as well if we decide to merge, mainly just me wanting to upload one overall safety score rather than 6 new numbers.

Reply:
right now since most models are a finetuned version of llama3, it can be simple to add the template [here](https://github.com/allenai/OE-Safety/blob/main/src/templates/single_turn.py), but if we want to use HF directly, that would require a few changes here and there. 
Averaging scores sounds good, we just need to ensure that we only average scores that measure the same thing.

Reply:
Oh looks like it already supports using the hf chat template! https://github.com/allenai/OE-Safety/blob/main/src/templates/single_turn.py#L260